tests/common.go

```diff
@@ -14075,6 +14079,113 @@ func GetVolumeNamefromPVC(namespace string) ([]string, error) {
 	return nil, fmt.Errorf("No PVCs found in namespace [%s]", namespace)
 }
 
+func DoVolumeQuorumValidation(volumeQuorumValidationStopSignal chan struct{}, vQuorumError *error) {
+	itr := 1
+	for {
+		select {
+		case <-volumeQuorumValidationStopSignal:
+			log.Infof("Exiting volume quorum validation routine")
+			return
+		default:
+			log.Infof("Volume quorum validation iteration: #%d", itr)
+			errorChan := make(chan error, 50)
+			ValidateVolumeQuorum(&errorChan)
+			close(errorChan) // Close the channel to signal completion
+			for err := range errorChan {
+				if err != nil {
+					*vQuorumError = multierr.Append(*vQuorumError, err)
+				}
+			}
+			if *vQuorumError != nil {
+				return
+			}
+			itr++
+			time.Sleep(10 * time.Second)
+		}
+	}
+}
+
+func ValidateVolumeQuorum(errChan ...*chan error) {
+	volIDs, err := Inst().V.ListAllVolumes()
+	if err != nil {
+		// add to errChan
+		err := fmt.Errorf("error listing volumes, Err: %v", err)
+		processError(err, errChan...)
+		return
+	}
+
+	for _, volID := range volIDs {
+		apiVol, err := Inst().V.InspectVolume(volID)
+
+		if err != nil {
+			err = fmt.Errorf("error inspecting volume [%s], Err: %v", volID, err)
+			processError(err, errChan...)
+			return
+		}
+
+		// check if volume replicas are on different nodes
+		// get all the nodes where replicas are present
+		if len(apiVol.ReplicaSets) == 0 {
+			err := fmt.Errorf("volume [%s] does not have any replicas", volID)
+			processError(err, errChan...)
+			return
+		}
+
+		replicaNodes := apiVol.ReplicaSets[0].Nodes
+
+		if apiVol.Status != opsapi.VolumeStatus_VOLUME_STATUS_UP {
+			// if it is repl-2 volume and replicas are on same node, do not fail the test
```

Comment:
nit: fix the comment; skip repl-1 volume validation

Reply:
done